### PR TITLE
Mitigate one kitchen overuse

### PR DESCRIPTION
**Pull request by hroncok**
This function converted a fixed unicode character to itself

Signed-off-by: Miro Hrončok <miro@hroncok.cz>

Disclaimer: This is untested.

### CODE REVIEW

File: bodhi/tests/server/test_mail.py [hunk 1]
```diff
@@ -90,7 +88,7 @@ def test_encoding(self):
 
         # Assemble the template for easier asserting.
         t = '\n'.join([l for l in t[0]])
-        self.assertTrue(converters.to_unicode('\xe7') in t)
+        self.assertTrue('\xe7' in t)
```
Comment (sebwoj) [inline bodhi/tests/server/test_mail.py]: I recommend `assertIn` here.

**Review by sebwoj — CHANGES_REQUESTED** [review]

**Comment by hroncok** [comment]
Should I worry about the CI? This could not have possibly caused breakage in integration tests.

File: bodhi/tests/server/test_mail.py (hunk 1 above)
Comment (hroncok) [inline bodhi/tests/server/test_mail.py]:
```suggestion
        self.assertIn('\xe7', t)
```

**Comment by hroncok** [comment]
Note: If I use the suggestions feature on GitHub, the DCO thing breaks. Off to a terminal.

**Comment by hroncok** [comment]
https://github.com/fedora-infra/bodhi/issues/3074

File: bodhi/tests/server/test_mail.py [hunk 2]
```diff
@@ -90,7 +88,7 @@ def test_encoding(self):
 
         # Assemble the template for easier asserting.
         t = '\n'.join([l for l in t[0]])
-        self.assertTrue(converters.to_unicode('\xe7') in t)
```
Comment (bowlofeggs) [inline bodhi/tests/server/test_mail.py:93]: Haha, kitchen is so bad.

**Review by bowlofeggs — APPROVED** [review]
Thanks for the patch! I'm going to squash it to one commit and merge.

**Pull request merged**